index.html

```diff
@@ -1419,6 +1421,17 @@ <h1>Glossary</h1>
                 </dd>
             <dt><dfn>hit test</dfn></dt>
                 <dd>The process by which a user agent determines a target element for a pointer event. Typically, this is determined by considering the pointer's location and also the visual layout of elements in a document on screen media.</dd>
+            <dt><dfn>measurable properties</dfn></dt>
+                <dd><p>Measurable properties represent values relating to pointer sensor data that is expressed using a real number or an
+                    integer from a large domain. For pointer events, <code>width</code>, <code>height</code>, <code>pressure</code>,
```

Comment:
what does "a large domain" mean exactly here, or does it really matter here?

Reply:
if something only had a handful of values (like...`button` and `buttons` for instance, which use a discrete set of integer values https://w3c.github.io/pointerevents/#button-states), it wouldn't be "measurable" in this definition, but countable/discrete

the domain/large domain tries to square the circle of "yes, tiltX/tiltY are integers, and yes they can only have a specific discrete number of values...but we do treat them as measurable because that number of values is so large"

Reply:
We could argue that the underlying data for tiltX/tiltY is a real number whereas buttons is inherently an integer value.

Reply:
perhaps...to square this particular circle (and remembering your original idea/mention  of "continuous" values) this could be modified along the lines of `Measurable properties represent values relating to continuous pointer sensor data`, or `Measurable properties represent continuous values relating to pointer sensor data`. thoughts? this would again clarify with button/buttons aren't there...

Reply:
done